### DISCUSSION TITLE
deleting a team

### DISCUSSION THREAD

**Discussion by xxxxasasxas** [Minestom/Minestom#1259] [Help] [answered]
how would I delete a team after i'm done using it?

**Comment by xxxxasasxas** [comment, marked as answer]
nvm found it, it didnt pop up for some reason

**Reply by emortaldev** [reply]
Edit the answer so others can know too :)

**Comment by xxxxasasxas** [comment]
All I did was `MinecraftServer.getTeamManager().deleteTeam(team)`. It wasn't popping up for some reason until I went into the actual team class and saw the method.
